### PR TITLE
added styles thumbnails for now

### PR DESCRIPTION
**Pull request by Khoidao55**

### CODE REVIEW

File: client/app/components/App.jsx [hunk 1]
```diff
@@ -34,7 +34,12 @@ const App = () => {
       .then((relatedProductsResponse) => {
         putInState.relatedProductIDs = relatedProductsResponse.data;
         return axios.get(`/reviews/meta/${putInState.id}`);
+        //return axios.get(`/products/${putInState.id}/styles`);
       })
+      // .then((productStylesResponse) => {
+      //   putInState.differentStyles = productStylesResponse.data;
+      //   return axios.get(`/reviews/meta/${putInState.id}`);
+      // })
```
Comment (tdagai) [inline client/app/components/App.jsx]: Why comment all this out?

Comment (Khoidao55) [reply client/app/components/App.jsx]: that was before we did our change together, i'll delete it on next pull

**Review by tdagai — COMMENTED** [review]
Take a look a the comment I left on App.jsx

**Review by zamrini — APPROVED** [review]
Looks good - let us know about the App.jsx get comment-out before merge please

**Pull request closed without merging**